style/main.css

```diff
@@ -0,0 +1,8 @@
+@tailwind base;
+@tailwind utilities;
+@tailwind components;
+
+* {
+    font-family: arial,sans-serif;
+}
+
```

Comment:
This font customization should be added to the tailwindcss config not in a CSS file